gulpfile.js

```diff
@@ -59,10 +60,18 @@ function inlineRegexSource() {
 function minifyJS() {
 	return [
 		inlineRegexSource(),
-		uglify()
+		uglify(),
 	];
 }
 
+function lint(cb) {
+	return pump([
+		src(['**/*.js', '!node_modules/**'], { base: './' }),
+		eslint(),
+		eslint.format(),
+		eslint.failAfterError(),
```

Comment:
Implementing a `failAfterWarningOrError` should be trivial looking at the [implementation of `failAfterError`](https://github.com/adametry/gulp-eslint/blob/master/index.js#L161).